app/assets/stylesheets/application.css.scss

```diff
@@ -6,3 +6,32 @@
 @import "neat";
 @import "base/base";
 @import "refills/flashes";
+
+.page-title {
+  font-size: 4em;
+}
+
+.products {
+  .title {
+    font-size: 3em;
+  }
+
+  .author {
+    font-size: 1.5em;
+    padding-left: 1em;
+  }
+
+  .blurb {
+    font-size: 1em;
+  }
+
+  li {
+    background-color: #f00;
```

Comment:
Color literals like `#f00` should only be used in variable declarations; they should be referred to via variable everywhere else.